File: vpe2020/2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int n;
    cin >> n;
    int arr[15];
    int tot = 0;
    for (int i = 0; i < n; i++) {
        cin >> arr[i];
        tot += arr[i];
    }
    tot /= n;
    int t = 0;
    for (int i = 0; i < n; i++) {
        if (tot != arr[i]) ++t;
    }
    cout << t;
}

File: ecoo 2020/2.py

for bob in range(int(input())):
    mp = {}
    lst = []
    for cat in range(int(input())):
        for dog in range(int(input())):
            inp = input().split()
            if inp[0] in mp:
                index = mp[inp[0]]
                lst[index].append([int(inp[1]),int(inp[2])])
            else:
                mp[inp[0]] = len(lst)
                lst.append([[int(inp[1]),int(inp[2])]])
    for i in lst:
        i.sort()
    tot = 0
    for puppy in range(int(input())):
        inp = input().split()
        arr = lst[mp[inp[0]]]
        quan = int(inp[1])
        ind = -1
        while quan > 0:
            ind += 1
            if quan > arr[ind][1]:
                tot += arr[ind][0]*arr[ind][1]
                quan -= arr[ind][1]
            else:
                tot += arr[ind][0]*quan
                quan = 0
    print(tot)

